certified-connectors/Data8/apiProperties.json

```diff
@@ -17,7 +17,8 @@
     },
     "iconBrandColor": "#1973AE",
     "capabilities": [],
+    "policyTemplateInstances": [],
```

Comment:
@mattybeard Since there are no policy templates defined, the property can be removed.